src/Node/ChildProcess.purs

```diff
@@ -237,7 +237,9 @@ spawnSync command args = (UnsafeCP.spawnSync command args) <#> \r ->
   , exitStatus: case toMaybe r.status, toMaybe r.signal of
       Just c, _ -> Normally c
       _, Just s -> BySignal s
-      _, _ -> unsafeCrashWith $ "Impossible: `spawnSync` child process neither exited nor was killed."
+      _, _ -> if isJust $ toMaybe r.error
+              then BySysError
+              else unsafeCrashWith $ "Impossible: `spawnSync` child process neither exited nor was killed."
```

Comment:
```suggestion
      _, _ ->
        if isJust $ toMaybe r.error then
          BySysError
        else
          unsafeCrashWith $ "Impossible: `spawnSync` child process neither exited nor was killed."
```

Reply:
Or option 2:

```suggestion
      _, _
        | isJust (toMaybe r.error) -> BySysError
        | otherwise -> unsafeCrashWith "Impossible: `spawnSync` child process neither exited nor was killed."
```

Reply:
Oh, I like this one! Let me try this

Reply:
Ah sorry, didn't see this message until now :smile: I think either way is fine though stylistically, and they should compile to pretty much the same JS.

Reply:
It's okay and thank you!